### DISCUSSION TITLE
Symfony Mercure

### DISCUSSION THREAD

**Discussion by jarzebowsky** [shinsenter/php#35] [Ideas]
Hey, perhaps this is somewhere but I could not be able to find it.

Is the project supporting the [Mercure](https://github.com/symfony/mercure)?

**Comment by shinsenter** [comment]
Hey @Jarzebowsky 

I think Symfony Mercure is more like a package for Symfony rather than a standalone framework. You could try using the [docker image for Symfony](https://docker.shin.company/symfony) to start a container, then use composer to install and use that package.

**Reply by jarzebowsky** [reply]
@shinsenter I did, but it's also have additional routes `./well-known/mercure` which is not supported by `phpfpm-nginx` or `symfony` image which is using the same.

**Reply by shinsenter** [reply]
Hey @Jarzebowsky 

I appreciate you raising this issue. Unfortunately I do not have enough context to offer a well-informed opinion at this time.

However, the Mercure documentation may provide some helpful guidance on project setup that could be worth exploring:
https://symfony.com/doc/current/mercure.html#installation.

Please let me know if reviewing that gives you any new insights into how to move forward. I'm happy to continue discussing options with you.

**Reply by jarzebowsky** [reply]
You were right as I was doing it in a rush it ended up being auto updated to `docker-compose.yaml` file. My problem is resolved as I switched to different service where I could install **mercure hub** next to symfony and connect to it successfully. Sorry for the rush from my side to open this up quickly before fully verifying the problem.
